src/base/bittorrent/sessionimpl.cpp

```diff
@@ -298,6 +298,20 @@ namespace
         return {};
     }
 #endif
+
+    constexpr lt::move_flags_t toNative(const MoveStorageMode mode)
+    {
+        switch (mode)
+        {
+        default:
```

Comment:
I would put an `Q_ASSERT(false);` under default case. Mainly to avoid forgetting adding new cases when `MoveStorageMode` has more options.

Reply:
> I would put an `Q_ASSERT(false);` under default case. Mainly to avoid mishandling new cases when `MoveStorageMode` has more options.

Done.